packages/cloudflare/src/pages-plugin.ts

```diff
@@ -23,7 +23,16 @@ export function sentryPagesPlugin<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Params extends string = any,
   Data extends Record<string, unknown> = Record<string, unknown>,
->(options: CloudflareOptions): PagesPluginFunction<Env, Params, Data, CloudflareOptions> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
l: If one can only access env variables in the function, should we update the JSDoc of this function?

Reply:
yes for sure, good call